wolox-books/src/styles.scss

```diff
@@ -0,0 +1 @@
+@import './scss/application';
```

Comment:
Add empty line here

Reply:
Ok, it changes a little, now we don't need application file, you can importe directly in this file 

@import 'equalizer';
@import 'form';
@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');